static/js/place/place_template.tsx

```diff
@@ -466,6 +466,150 @@ class ChildPlace extends Component<ChildPlacePropType, unknown> {
   }
 }
 
+interface ChartBlockPropType {
```

Comment:
Nit: can we move the chart related components to a new file?

Reply:
we can also move after, in the interest of time.

Reply:
sg, added TODO